services/mailservice/mail.go

```diff
@@ -62,15 +80,15 @@ type SmtpConnectionInfo struct {
 
```

Comment:
We can actually remove the struct declaration above. L69.

Reply:
:facepalm: